code/modules/admin/smites/bread.dm

```diff
@@ -1,12 +1,15 @@
 #define BREADIFY_TIME (5 SECONDS)
 
-/// Turns the target into bread
+/// Turns the target into bread (Bongbread if they are Bong Bong)
 /datum/smite/bread
 	name = "Bread"
 
 /datum/smite/bread/effect(client/user, mob/living/target)
 	. = ..()
-	var/mutable_appearance/bread_appearance = mutable_appearance('icons/obj/food/burgerbread.dmi', "bread")
```

Comment:
by the way, with how DM compiles you need to have var/mutable_appearance/bread_appearance always defined as a variable before code uses it, even if its defined by the if()else() check that always gives an output
in other words, keep the line above as
```
var/mutable_appearance/bread_appearance
``` and the code should be good

Reply:
So like that? Or did I misunderstand?

Reply:
yep, but now you have to not re-define the same variable. So remove the other instances of `var/mutable_appearance/bread_appearance` replacing them with just `bread_appearance`

Reply:
Okay, there we go. Hope that is good?

Reply:
Yep, should work perfectly well